src/Hl7.Fhir.Base/Serialization/BaseFhirJsonPocoSerializer.cs

```diff
@@ -64,7 +64,7 @@ public BaseFhirJsonPocoSerializer(FhirRelease release, FhirJsonPocoSerializerSet
         /// Serializes the given dictionary with FHIR data into Json.
         /// </summary>
         public void Serialize(IReadOnlyDictionary<string, object> members, Utf8JsonWriter writer) =>
-            serializeInternal(members, writer, skipValue: false);
+            serializeInternal(members, writer, skipValue: false, filter: null);
```

Comment:
When calling the serializeInternal the "filter: null" is unnecessary as it's already the default value for the parameter, you can leave this out.